sankey-trash/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Wrapper-starter Visual Vocabulary</title>
+
+    <script src="//unpkg.com/@financial-times/d3-bootloader@^1.6.0" async></script>
+    <link rel="stylesheet" href="https://ig.ft.com/visual-vocabulary/styles.css"/>
+    <link rel="stylesheet" href="//www.ft.com/__origami/service/build/v2/bundles/css?modules=o-fonts@^2.2.0" />
+</head>
+<body>
+
+    <h1>Sankey</h1>
+    <p>A useful way of showing non-cyclical transfer from one entity to another</p>
+    <!-- <p><b>Some useful settings in the index.js</b>
+        <ul>
+            <li><b>yMin</b> and <b>yMax</b> – use these to set the lower and upper values of the y-axis</li>
+            <li><b>numTicksy</b> – set the number of scale lines on the y-axis</li>
+            <li><b>annotate</b> – set to 'true' will display annotations in your dataset's 'annotate' column</li>
+            <li><b>highlightNames</b> – put the column names of the data you want to highlight, all other series will be muted</li>
+            <li><b>interval</b> – options available are "century","jubilee" (50 years), decade", "lustrum" (5 years), "years","months","days"</li>
+        </ul>
+    </p> -->
+     <p><b>Column headings in the csv</b>
```

Comment:
Sorry, should've caught this — this should probably be updated to source/target/value.